services/ingestion/etl/wifi4eu/xls/src/lib/getCountryCode.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * Retrieve a country code based on non-standartized input values.
+ *
+ * @memberof Wifi4EuXlsTransform
+ * @param {Object} country Country in the form of 'БЪЛГАРИЯ (BULGARIA)', 'ΕΛΛΑΔΑ (ELLADA)' and similar.
+ * @returns {String} ISO ALPHA-2 code  used by i18n-iso-countries in the enrichment manager.
+ */
+export default country => {
+  switch (country) {
+    case 'БЪЛГАРИЯ (BULGARIA)':
+      return 'BG';
+    case 'HRVATSKA':
+      return 'HR';
+    case 'FRANCE':
+      return 'FR';
+    case 'ESPAÑA':
+      return 'ES';
+    case 'ITALIA':
+      return 'IT';
+    case 'DEUTSCHLAND':
+      return 'DE';
+    case 'SLOVENIJA':
+      return 'SI';
+    case 'POLSKA':
+      return 'PL';
+    case 'MALTA':
+      return 'MT';
+    case 'PORTUGAL':
+      return 'PT';
+    case 'ÖSTERREICH':
+      return 'AT';
+    case 'ΕΛΛΑΔΑ (ELLADA)':
+      return 'GR';
+    case 'NEDERLAND':
+      return 'NL';
+    case 'SVERIGE':
+      return 'SE';
+    case 'SLOVENSKO':
+      return 'sk';
```

Comment:
why is this one lowercase?

Reply:
Was a test one whether it matters, but it doesn't, thanks for noticing!